float32 pn
float32 pe
float32 pd
float32 phi
float32 theta
float32 chi
float32 azimuth
float32 elevation

sensor_msgs/CompressedImage image
